kernel/syscalls/README.md

```diff
@@ -1,18 +1,58 @@
 # System calls
 
+System call commonly abbreviated to syscall makes an entry point to execute a specific user program's request to a service from the kernel. The operating system kernel runs in privileged mode to protect sensitive software or hardware parts from other software components. User application executing in unprivileged mode does not have access to protected data. Performing a hardware interrupt or conducting a trap handled by the kernel, the user application can obtain sensitive data from the kernel, e.g. information about all processes running in the operating system.
 
+## Declaration and definition
 
+In Phoenix-RTOS declarations and definitions of the system calls are located in a `libphoenix` library. The list of the all system calls is placed in `phoenix-rtos-kernel/include/syscalls.h` grouped by the categories.
 
+The system calls declaration should be placed in the appropriate header file in `libphoenix` library, referring to the syscall's category.
```

Comment:
```suggestion
System call declarations should be placed in the an appropriate header file in the `libphoenix` standard library, referring to the syscall's category.
```
prototypes really, not declarations. Also, maybe we shouldn't mix "system call" with "syscall"